Common/DataModel/T0infoTable.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+#ifndef O2_T0_INFO_TABLE_H_
+#define O2_T0_INFO_TABLE_H_
+
+#include "Framework/AnalysisDataModel.h"
+
+namespace o2::aod
+{
+namespace infoFT0
+{
+DECLARE_SOA_COLUMN(T0A, t0A, float);           //! Collision time A-side, corrected with primary vertex
+DECLARE_SOA_COLUMN(T0C, t0C, float);           //! Collision time A-side, corrected with primary vertex
+DECLARE_SOA_COLUMN(T0AC, t0AC, float);         //! Collision time (A+C)/2
```

Comment:
No, I calculate (T0A+T0C)/2  at line 57, then, afterwards, at  line 59
I add correction to one side and subtract from the opposite.
So, we got 3 collision times for TOF: 1. the best one (T0A+T0C)/2, 2. T0A, 3. T0C. All three are not depend on vertex position: 1. - naturally, 2&3  were corrected

Reply:
Alla, it is the same isn't it?

AC=(A+C)/2
A'=A+V
C'=C-V
--> (A'+C')/2 = (A+V+C-V)/2 = (A+C)/2 = AC

What am I overlooking?

Reply:
A and C should be used separately. When both are exist - AC using.  A or  C are used in low multiplicity events when only one of them exist. A and C has worse resolution then AC.

Reply:
But would one use `AC` if only `A` or `C` exist?
My point is that it is mathematically identical if `ora` and `orc` are `true` in your code.
Could you tell me me where is the mistake in my calculation? Otherwise I fail to understand your point.

Reply:
AC is collision time with very good resolution. But in events with low multiplicity (due to acceptance) could be that only A or only C exits. But average time on each side depend on vertex, yes? So to use A (or C)   as collision time we need to remove vertex dependence. Resolution of these collision time would be much worse, but still usable. In Run2 for p-p AC with resolution ~50ps were exist in 50% events, only A (with resolution ~75ps) added ~10% events , only C (with resolution ~70ps) provided additional 15%. It was rather important for TOF.

Reply:
Could you tell me me where is the mistake in my calculation? Otherwise I fail to understand your point.

Reply:
Yes, is it possible to use primary vertex in dynamic column? I think AC and vertex could be dynamic for sure.